src/components/common/meta/ArticleMeta.js

```diff
@@ -72,44 +109,7 @@ const ArticleMetaGhost = ({ data, settings, canonical }) => {
 
                 {settings.twitter && <meta name="twitter:site" content={`https://twitter.com/${settings.twitter.replace(/^@/, ``)}/`} />}
                 {settings.twitter && <meta name="twitter:creator" content={settings.twitter} />}
-                <script type="application/ld+json">{`
-                    {
-                        "@context": "https://schema.org/",
-                        "@type": "Article",
-                        "author": {
-                            "@type": "Person",
-                            "name": "${author.name}",
-                            ${author.image ? author.sameAsArray ? `"image": "${author.image}",` : `"image": "${author.image}"` : ``}
```

Comment:
This conditional doesn't make a lot of sense. Why is `sameAsArray` being checked here? Is there some logic related to `sameAs` which is missing?

Reply:
The difference is simply the `,` which has to be appended when `author.sameAsArray` is populated and will follow after. In case it's not populated, the `author.image` property will be the last property in this object.

Reply:
This had be done **after** it provenly failed in the [Google Structured Data testing tool](https://search.google.com/structured-data/testing-tool/u/0/)

Reply:
Ahh I see, that just goes away now.

Reply:
Exactly! Much cleaner now 😺